Spryker analytics (#2880)
spryker/spryker-docs: docs/pbc/all/business-intelligence/202410.0/amazon-quicksight-third-party-integration/set-up-amazon-quicksight.md

---
title: Set up Amazon QuickSight
description: First steps to set up Amazon QuickSight
template: concept-topic-template
last_updated: Oct 24, 2024
---

This document describes how to set up Amazon QuickSight.

1. [Install the Amazon QuickSight feature]().

>   At this stage, the **Analytics** section is displayed in the Back Office, but the functionality is not available.

Reviewer: please remove it: "At this stage, the Analytics section is displayed in the Back Office, but the functionality is not available." and make this point number 2. The first should be "To request Business Intelligence powered by Amazon QuickSight…"

> 2. To request Amazon QuickSight analytics, contact your Customer Success Manager.

Reviewer: ```suggestion
2. To request Business Intelligence powered by Amazon QuickSight, contact your Customer Success Manager at https://now.spryker.com/contact-analytics.
```

Reviewer: we need to use the official SKU name here "Business Intelligence powered by Amazon QuickSight"

>   After the contract is signed, we provision Amazon Quicksight for your project.

Reviewer: ```suggestion
  After the contract is signed, we will activate Amazon QuickSight for your project.
```

3. In the Back Office, go to **Analytics**.

> 4. On the **Analytics** page, click **Enable Analytics**.

Reviewer: ```suggestion
4. On the **Analytics** page, click **Enable Analytics**. This reopens the **Analytics** page with a progress message displayed. 
```

{% info_block warningBox "" %}

* The user who enables analytics is assigned as an owner of all the default assets, such as data sets, analyses, and dashboards. You can transfer ownership later.

> * The user who enables analytics is automatically assigned with the Author role.

Reviewer: ```suggestion
* The user who enables analytics must have an assigned Author role.
```

{% endinfo_block %}  

> 5. On the **Enable Analytics** page, click **Enable Analytics**.

Reviewer: remove the 5th point and make point 6 instead
